app/src/main/java/com/andrew/tasky/presentation/adapters/PhotoItemAdapter.kt

```diff
@@ -0,0 +1,47 @@
+package com.andrew.tasky.presentation.adapters
+
+import android.net.Uri
+import android.view.LayoutInflater
+import android.view.ViewGroup
+import androidx.recyclerview.widget.RecyclerView
+import com.andrew.tasky.databinding.ItemPhotoAdapterCardBinding
+import com.andrew.tasky.util.AgendaItemDetailFragmentCommunicationWithRV
+
+class PhotoItemAdapter(
+    private var photos: List<Uri>,
+    private val listener: AgendaItemDetailFragmentCommunicationWithRV
```

Comment:
Instead of such a long and rather less descriptive name, I'd just use simple Kotlin lambdas. For example:
```
private val openPhoto: (index: Int) -> Unit
```